controllers/src/main/scala/cromwell/pipeline/controller/AkkaHttpClient.scala

```diff
@@ -0,0 +1,85 @@
+package cromwell.pipeline.controller
+
+import java.nio.charset.Charset
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.headers.RawHeader
+import akka.http.scaladsl.model.{ HttpMethods, HttpRequest, HttpResponse }
+import akka.http.scaladsl.{ Http, HttpExt }
+import akka.stream.ActorMaterializer
+import akka.util.ByteString
+import cromwell.pipeline.service.{ HttpClient, Response }
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class AkkaHttpClient extends HttpClient {
+  private implicit val actorSystem: ActorSystem = ActorSystem()
+  private implicit val materializer: ActorMaterializer = ActorMaterializer()
+
+  def http: HttpExt = Http()
+
+  override def get(url: String, params: Map[String, String] = Map(), headers: Map[String, String] = Map())(
+    implicit ec: ExecutionContext
+  ): Future[Response] = {
+    val futureResponse: Future[HttpResponse] = http.singleRequest(
+      HttpRequest(method = HttpMethods.GET, uri = queryBuilder(url, params)).withHeaders(parseHeaders(headers))
+    )
+    responsify(futureResponse)
+  }
+
+  override def post(
+    url: String,
+    params: Map[String, String] = Map(),
+    headers: Map[String, String] = Map(),
+    payload: String = ""
```

Comment:
Empty string is never a good thing. 
@myazinn come here and tell them the truth, please. 

Also, it's a good idea to make slides for newcomers like "5 simple things you should never do".

Reply:
Yep, you should never use such placeholders.
If you think that there are cases when you don't need this parameter (or a client may not have it), then you should take `Option[String]` as a parameter.
Otherwise, this parameter may be passed from one method to another many time causing a lot of corrupted data. Fixing this is usually a nightmare :)

Reply:
Get it, thanks, remove empty string. But I can not change trait, where payload is a String, not Optional[String]